lib/auth/tun.go

```diff
@@ -590,8 +601,19 @@ func (c *TunClient) Close() error {
 
 // GetDialer returns dialer that will connect to auth server API
 func (c *TunClient) GetDialer() AccessPointDialer {
-	return func() (net.Conn, error) {
-		return c.Dial(c.authServers[0].AddrNetwork, "accesspoint:0")
+	return func() (conn net.Conn, err error) {
+		for _, a := range c.getAuthServers() {
```

Comment:
Just to clarify, we don't need this outer loop, as it does not really use the servers addresses anywhere, but you can keep inner attempts loop